index.html

```diff
@@ -879,7 +965,7 @@ <h3>Controllers</h3>
             <p class="note" title="Authentication versus Authorization">
 Note that the definition of [=authentication=] is different from the definition
 of [=authorization=]. Generally speaking, [=authentication=] answers the
-question of "Who is this?" while [=authorization=] answers the question of "Are
+question of "Do we know who this is?" while [=authorization=] answers the question of "Are
 they allowed to perform this action?". The `authentication` property in this
```

Comment:
```suggestion
question of "Do we know who this is?" while [=authorization=] answers the question of "Do we know whether
they are allowed to perform this action?". The `authentication` property in this
```

Reply:
In this case, I don't think the question is "do we know" but rather "are they ACTUALLY allowed".

To wit: the verifier may not know before doing the analysis, especially in cases of object capabilities and bearer tokens.

So, it isn't "answers the question of 'Do we know...?'" but rather "answers the question of 'can we determine if they are allowed'", which to me is just a wordier way to say "answers the question of 'are they allowed...?'"

Reply:
The question of "are they ACTUALLY allowed" cannot be answered, unless we have some affirmative (or negative) statement to that effect. If we have no specific statement, then the Open World assumption means we can only say, "we don't know".